libnavigation-ui/src/test/java/com/mapbox/navigation/ui/NavigationViewModelTest.java

```diff
@@ -85,48 +71,16 @@ public void updateRoute_navigationIsNotUpdatedWhenChangingConfigurations() {
     DirectionsRoute route = mock(DirectionsRoute.class);
     MapConnectivityController mockedConnectivityController = mock(MapConnectivityController.class);
     MapOfflineManager mapOfflineManager = mock(MapOfflineManager.class);
-    NavigationViewRouter router = mock(NavigationViewRouter.class);
+    NavigationViewOptions mockNavigationViewOptions = mock(NavigationViewOptions.class);
     NavigationViewModel viewModel = new NavigationViewModel(application, navigation, mockedConnectivityController,
-            mapOfflineManager, router);
+            mapOfflineManager, mockNavigationViewOptions);
     viewModel.onDestroy(true);
 
-    viewModel.updateRoute(route);
+    //viewModel.retrieveRoute().
```

Comment:
What are we testing here?

Reply:
The functionality is removed as well. Forget to remove the test.
cc @Guardiola31337